config/environments/development.rb

```diff
@@ -113,5 +113,5 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
-  config.active_storage.service = :amazon
+  config.active_storage.service = :local
```

Comment:
The other team working on the Custom Landing Page editor might be relying on the storage being set to amazon for proper testing during development. I'd suggest keeping it as `:amazon` for now.

Reply:
Yep, but when I setup the application, I get error without configure Amazon. Therefore, I have configured on local storage.